docs/source/guide/index.rst

```diff
@@ -341,9 +369,10 @@ If you choose ``-b/--bootstrap`` it will automatically send the bootstrap to
 
 If you did the ceremony in a disconnected computer:
 Using another computer with access to ``repository-service-tuf-api``
-1.  Get the generated ``payload.json`` (or the custom name you chose)
-2.  Install ``repository-service-tuf``
-3.  Run ``rstuf admin ceremony -b [-u filename]``
+
+  1.  Get the generated ``payload.json`` (or the custom name you chose)
+  2.  Install ``repository-service-tuf``
+  3.  Run ``rstuf admin ceremony -b [-u filename]``
```

Comment:
Isn't the `u` option required to upload it online?
As it seems that you document it as optional or am I wrong?